Makefile

```diff
@@ -6,7 +6,8 @@ NETLIFY_FUNC      = $(NODE_BIN)/netlify-lambda
 # but this can be overridden when calling make, e.g.
 # CONTAINER_ENGINE=podman make container-image
 CONTAINER_ENGINE ?= docker
-CONTAINER_IMAGE   = kubernetes-hugo
+IMAGE_VERSION=$(shell scripts/hash-files.sh Dockerfile | cut -c 1-12)
```

Comment:
technically the makefile should maybe also change this but that will probably mostly create false version changes.
we're taking the first 12 characters of the hash here, which seems sufficient.

it no longer requires manually bumping an image version. any dockerfile change will generate a new hash.

Reply:
I would include the Makefile hash here. We rarely change the Makefile (recent history is an exception).

Reply:
I think the hash of the Dockerfile will be sufficient indeed. Contributors are running the container using the Makefile, meaning they already have the correct Makefile. It's the container image they need to pull.

Reply:
1. Make won't know that a container image built with a different `Makefile` is stale.
2. Some people don't have access to the official Kubernetes container image repository to pull it, so they need to build it. Also, some people use a local Hugo for previewing and that is still a valid option. `make` wraps the commands to do this.

Reply:
Both valid points. Thanks

Reply:
I think with a little work and a placeholder output file we can actually use these to automatically pull / rebuild when the hash changes. We can also probably attempt a pull and then fallback to building automatically.